docs/configuration-file/comprehensive-example.md

```diff
@@ -0,0 +1,71 @@
+### Comprehensive Example
+
+The following is a comprehensive configuration file found [here](https://github.com/ALSchwalm/transient/blob/master/test/config-files/comprehensive-config).
+
+```
+# A Transient config file
+
+#
+# Transient arguments are set in a similar manner compared to the command line.
+# A few differences exist:
+#
+#   1. All values are set using an equal sign:
+#
+#          shutdown-timeout = 20
+#
+#
+#   2. Strings are encapsulated with double quotes
+#
+#          name = "vm_name"
+#
+#
+#   3. Flags are set to Booleans:
+#
+#          prepare-only = true
+#          ssh-console = false
+#
+#
+#   4. Lists are encapsulated by square brackets:
+#
+#          shared-folder = [
+#              "/path/on/host:/path/on/guest",
+#              "/path/on/host2:/path/on/guest2"
+#          ]
+#
+
+[transient]
+
+copy-in-before   = []
+copy-out-after   = []
+copy-timeout     = 0
+image            = ["centos/7:2004.01"]
+image_backend    = "$HOME/.local/share/transient/backend"
+image_frontend   = "$HOME/.local/share/transient/frontend"
+name             = "test-vm"
+prepare-only     = true
+qmp-timeout      = 10
+shared-folder    = []
+shutdown-timeout = 20
+ssh-bin-name     = "/sbin/ssh"
+ssh-command      = "whoami"
+ssh-console      = false
+ssh-port         = 22
```

Comment:
This should probably be a high port, just so it is clear to people that 22 is _not_ the default.

Reply:
From https://transient.readthedocs.io/en/latest/cli/run/:
`-ssh-port PORT: Connect to port PORT on the virtual machine for SSH instead of the default 22.`

Or am I grossly misunderstanding something?